app/src/org/commcare/android/util/CommCareUtil.java

```diff
@@ -87,19 +82,6 @@ public static int countPreds(TreeReference reference) {
         return preds;
     }
     
-    public static void printInstance(String instanceRef) {
-        try {
-            ByteArrayOutputStream bos = new ByteArrayOutputStream();
-            DataModelSerializer s = new DataModelSerializer(bos, new CommCareInstanceInitializer(null));
-            
-            s.serialize(new ExternalDataInstance(instanceRef,"instance"), null);
-            Log.d(TAG, new String(bos.toByteArray()));
-        } catch (IOException e) {
-            // TODO Auto-generated catch block
-            e.printStackTrace();
-        }
-    }
-
```

Comment:
I often will use this when I am debugging, and would kind of prefer to keep it somewhere accessible if possible.